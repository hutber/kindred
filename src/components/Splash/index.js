import React from 'react';
//import NavLink from '../shared/nav/NavLink';
import { NavLink } from 'react-router-dom'
import Logo from '../shared/index/Logo';
import * as styles from './style.css';

class Splash extends React.Component {
  render(){
    return (
      <div className={styles.indexAppContent}>
        <NavLink to="/home"  className={styles.index}>
          <Logo />
        </NavLink>
      </div>
    );
  }
}

export default Splash;
