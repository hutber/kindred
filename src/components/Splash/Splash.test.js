import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Splash from './Splash';

it('Splash page is rendered', () => {
	const result = shallow(
		<Splash />,
	);

	expect(shallowToJson(result)).toMatchSnapshot();
});
