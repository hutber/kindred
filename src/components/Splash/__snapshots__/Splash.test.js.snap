// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</Splash> We see the Splash Page with logo 1`] = `
<div
  className={undefined}
>
  <NavLink
    activeclassName="active"
    ariaCurrent="true"
    className={undefined}
    to="/home"
  >
    <Logo />
  </NavLink>
</div>
`;
